<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>1151</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.085, y1:1, x2:1, y2:0, stop:0 rgba(192, 192, 192, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>-10</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="cad_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Cadastrar Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cadcar_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Cadastrar Carro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loc_car">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Locar Carro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="comprar_car">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Comprar Carro</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Selecione uma opção</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>388</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Gerenciar Carros</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ger_cli">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>450</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
color: rgb(0, 0, 0);
border-radius:7;</string>
      </property>
      <property name="text">
       <string>Gerenciar Clientes</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>831</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delet_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>440</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:7;</string>
     </property>
     <property name="text">
      <string>DELETAR CARRO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_btn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>440</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:7;</string>
     </property>
     <property name="text">
      <string>EDITAR CARRO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="voltar_gerenciar">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>500</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:7;
</string>
     </property>
     <property name="text">
      <string>VOLTAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>440</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:7;</string>
     </property>
     <property name="text">
      <string>RESETAR TEMPO</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1137</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
